src/lsm/tree_fuzz.zig

```diff
@@ -645,9 +645,14 @@ const Model = struct {
     table_usage: TableUsage,
 
     fn init(table_usage: TableUsage) !Model {
+        const model_node_count = std.math.divCeil(
+            u32,
+            events_max * @sizeOf(Value),
+            NodePool.node_size,
+        ) catch unreachable;
```

Comment:
Fyi we have a `stdx.div_ceil` that doesn't return an error. (It asserts that the denominator is nonzero.)

Reply:
Ahhhh, I had a vague recollection of that, but grepped for `divCeil` which didtt' help :rofl: